components/resource_metering/Cargo.toml

```diff
@@ -35,6 +29,7 @@ slog-global = { version = "0.1", git = "https://github.com/breeswish/slog-global
 futures = "0.3"
 pdqselect = "0.1"
 fail = "0.4"
+thread-id = "4.0.0"
```

Comment:
dependency leak?

Reply:
I use crate `thread-id` in summary module (non-test) to get the unique id of the thread, which is cross-platform by default. In this case we only need a unique id, without any semantics.